docs/structure-of-a-contract.rst

```diff
@@ -8,6 +8,19 @@ Structure of a Contract
 Contracts in Vyper are contained within files, with each file being one smart-contract.  Files in Vyper are similar to classes in object-oriented languages.
 Each file can contain declarations of :ref:`structure-state-variables` and :ref:`structure-functions`.
 
+.. _structure-versions:
+
+Versions
+========
+
+Vyper supports version pragma which is used to reject being compiled with future compiler versions that might introduce incompatible changes.
+
+::
+
+  #@version0.1.13b
```

Comment:
just curious about my question here: https://github.com/ethereum/vyper/issues/1449#issuecomment-546710359

I would rather document it as this (if it works)
```suggestion
  # @version 0.1.13b
```

Reply:
yes `#@version` should parse fine